test/tstLinearBVH.cpp

```diff
@@ -740,6 +740,10 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(callback_with_attachment, DeviceType,
   }
 }
 
+// FIXME temporary workaround bug in HIP-Clang (register spill) and SYCL
+#if defined(KOKKOS_ENABLE_HIP) || defined(KOKKOS_ENABLE_SYCL)
+BOOST_TEST_DECORATOR(*boost::unit_test::expected_failures(5))
```

Comment:
How did you come up with that number?  I'd rather look into upgrading to ROCm 4.0

Also we need to investigate a bit more the SYCL issues.

Reply:
This is the number of calls to `BOOST_TEST` in this test. It could be lower than that, but I see no point trying to be precise, as for me the test is dead anyway in those configurations.